Sources/NIOEmbedded/Embedded.swift

```diff
@@ -130,7 +130,7 @@ public final class EmbeddedEventLoop: EventLoop, CustomStringConvertible {
 
     public let description = "EmbeddedEventLoop"
 
-    #if canImport(Darwin) || canImport(Glibc) || canImport(Musl) || canImport(Bionic)
+    #if canImport(Darwin) || canImport(Glibc) || canImport(Musl) || canImport(Android)
```

Comment:
The two conditionals are interchangeable, just updating this to match the actual import above.